view/src/components/PDF/PDF.js

```diff
@@ -1,48 +1,89 @@
 import React from "react";
 import "./PDF.css";
 import jsPDF from "jspdf";
-// import "jquery";
+import Project from "../../../../models/Project.js";
```

Comment:
I'm getting an error due to this line. It wouldn't work in production anyway because relative filepaths will be meaningless after webpacking. Try declaring a variable that looks like a sample project object and render based of that. Once the React is working, all we need to do is pass props as an argument instead of your sample project.

Reply:
Okay.  Imported dummy project object model.  Should line up.